pkg/segment/query/segquery.go

```diff
@@ -429,7 +429,9 @@ func GetNodeResultsForSegmentStatsCmd(queryInfo *QueryInformation, sTime time.Ti
 			ErrList: []error{err},
 		}
 	}
-	aggMeasureRes, aggMeasureFunctions, aggGroupByCols, _, bucketCount := allSegFileResults.GetSegmentStatsResults(0, true)
+
+	humanizeValues := !config.IsNewQueryPipelineEnabled()
+	aggMeasureRes, aggMeasureFunctions, aggGroupByCols, _, bucketCount := allSegFileResults.GetSegmentStatsResults(0, humanizeValues)
```

Comment:
Why don't we want to humanize?

Side note: it's a little odd that this change is in this PR, but that's ok

Reply:
This is when first stats is there like: `* | stats ...`. And you have something after that: `* | stats count | eval | ...` and more pipes, then if the values are humanized then we won't be able to perform subsequent command operations properly.